synapse/tests/test_lib_storm.py

```diff
@@ -550,6 +550,49 @@ class StormTest(s_t_utils.SynTest):
             q = 'inet:ipv4=1.2.3.4 | tee'
             await self.asyncraises(s_exc.StormRuntimeError, core.nodes(q))
 
+            # Runtsafe tee
+            q = 'tee { inet:ipv4=1.2.3.4 } { inet:ipv4 -> * }'
+            nodes = await core.nodes(q)
+            self.len(2, nodes)
+            exp = (
```

Comment:
can we use set comparisons here instead of sortedeq? we have very little use of sortedeq in mainline and keeps things consistent.